daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/parsers/LayeredSequenceParser.scala

```diff
@@ -54,6 +55,8 @@ class LayeredSequenceParser(
     } catch {
       case le: LayerNotEnoughDataException =>
         PENotEnoughBits(state, le.schemaFileLocation, le.dataLocation, le.nBytesRequired * 8, MaybeULong.Nope)
+      case e: Exception =>
+        throw LayerExecutionException(s"Unexpected exception in layer transformer ${layerTransformer.layerName}: $e", e)
```

Comment:
Add '...' around the layer name so that it is distinguished from the text of the message.

Reply:
Updated to place quotes around the layer name